src/app/shared/shared.module.ts

```diff
@@ -1,17 +1,33 @@
 import {NgModule} from '@angular/core';
 import {CommonModule} from '@angular/common';
 import {DirectiveModule} from './directives/directive.module';
+import {OrderByPipe, RoundPipe} from './pipes';
+import {BrowserModule} from '@angular/platform-browser';
+import {FormsModule, ReactiveFormsModule} from '@angular/forms';
 
 @NgModule({
   imports: [
+    BrowserModule,
+    FormsModule,
+    ReactiveFormsModule,
     CommonModule,
     DirectiveModule
   ],
-  declarations: [],
+  declarations: [
+    OrderByPipe,
+    RoundPipe,
```

Comment:
You can create and import PipeModule instead. The same as you did with DirectiveModule.

Reply:
Done